airflow/providers/microsoft/azure/operators/asb.py

```diff
@@ -206,6 +208,120 @@ def execute(self, context: "Context") -> None:
         hook.delete_queue(self.queue_name)
 
 
+class AzureServiceBusTopicCreateOperator(BaseOperator):
+    """
+    Create an Azure Service Bus Topic under a Service Bus Namespace by using ServiceBusAdministrationClient
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:AzureServiceBusTopicCreateOperator`
+
+    :param topic_name: Name of the topic.
+    :param default_message_time_to_live: ISO 8601 default message time span to live value. This is
```

Comment:
It would be nice to have hanging indents for the param descriptions for consistency with the code base and improved readability, but you could also consider this a nit. Up to you.